src/multicluster/extensions.ts

```diff
@@ -344,6 +359,44 @@ export const extensions: EncodedExtension[] = [
     },
     type: 'acm.resource/route',
   } as EncodedExtension<ResourceRoute>,
+
+  {
+    properties: {
+      component: {
+        $codeRef: 'Checkups',
+      },
+      path: [
+        '/k8s/all-clusters/all-namespaces/checkups',
+        '/k8s/cluster/:cluster/all-namespaces/checkups',
+        '/k8s/cluster/:cluster/ns/:ns/checkups',
+      ],
+    },
+    type: 'console.page/route',
+  } as EncodedExtension<RoutePage>,
+  {
+    flags: {
+      required: ['KUBEVIRT_DYNAMIC'],
+    },
+    properties: {
+      component: {
+        $codeRef: 'CheckupsSelfValidationForm',
+      },
+      path: ['/k8s/cluster/:cluster/ns/:ns/checkups/self-validation/form'],
```

Comment:
the checkups use heavily paths for navigation
1. both paths should be in sync - simplest solution is to extract common parts to const
2. we will need to test the navigation twice CC: @gouyang